content/blog/2019-08-15-new-react-devtools.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Introducing the New React DevTools"
+author: [bvaughn]
+---
+We are excited to announce a new release of the React Developer Tools, available today in Chrome, Firefox, and (Chromium) Edge!
+
+## What's changed?
+
+A lot has changed in version 4!
+At a high level, this new version should offer significant performance gains and an improved navigation experience.
+
```

Comment:
Just one screenshot or a gif here, pretty please? I need to see what I'm gonna be using.

Reply:
<img width="761" alt="Screen Shot 2019-08-15 at 10 44 41 AM" src="https://user-images.githubusercontent.com/29597/63114449-b6172a00-bf49-11e9-8b25-43d54e5fb07a.png">